build.gradle

```diff
@@ -24,4 +25,9 @@ patchPluginXml {
     changeNotes """
       Add change notes here.<br>
       <em>most HTML tags may be used</em>"""
+}
+
+checkstyle {
+    configFile = file('/google_checks.xml')
+    maxWarnings = 0
```

Comment:
This disallows warnings i.e. minor crimes against style. Our code must be perfect(?)